lib/src/main/java/im/status/hardwallet_lite_android/io/APDUResponse.java

```diff
@@ -37,7 +37,13 @@ public boolean isOK() {
         return this.sw == SW_OK;
     }
 
-    public APDUResponse checkOK() throws APDUException {
+    public APDUResponse checkOK(int... codes) throws APDUException {
```

Comment:
Maybe is more flexible to have this function as checkSW and leave checkOK() as a call to checkSW(0x9000). This would also avoid the need to check 0x9000 as a special case.

Reply:
cool, I added checkSW